src/decaylanguage/data/decfile.lark

```diff
@@ -4,24 +4,32 @@
 // or https://github.com/scikit-hep/decaylanguage for details.
 
 start : _NEWLINE* (line _NEWLINE+)+ ("End" _NEWLINE+)?
-?line : define | pythia_def | jetset_def | model_alias | alias | chargeconj | commands | decay | cdecay | copydecay | setlspw
+?line : define | particle_def | pythia_def | jetset_def | ls_def | model_alias | alias | chargeconj | commands | decay | cdecay | copydecay | setlspw | setlsbw | changemasslimit
 
-pythia_def : "PythiaBothParam" LABEL ":" LABEL "=" (LABEL | SIGNED_NUMBER)
+pythia_def : ("PythiaBothParam" | "PythiaAliasParam") LABEL ":" LABEL "=" (LABEL | SIGNED_NUMBER)
 
 jetset_def : "JetSetPar" LABEL "=" SIGNED_NUMBER
 
+ls_def : ("LSFLAT" | "LSNONRELBW") label
```

Comment:
Can you add a comment line for what this is? I know I could do with more doc in this file. We can improve as we go, and this one here is not an obvious label.